volatility3/framework/plugins/linux/sockstat.py

```diff
@@ -548,7 +548,7 @@ def list_sockets(
             except AttributeError:
                 netns_id = NotAvailableValue()
 
-            yield task, netns_id, fd_num, family, sock_type, protocol, sock_fields
```

Comment:
`list_sockets` is an expose plugin, so this API is changing in a non-backwards compatible way, meaning the plugin needs a MAJOR version bump, and please verify that any consumers of this plugin (within the core codebase) have their dependencies bumped and are tested to work with the new return value.  At the moment it doesn't look like any other plugins depend on it, so it should be fine to just bump it to `_version = (2, 0, 0)`...